src/sentry/static/sentry/app/views/performance/transactionSummary/charts.tsx

```diff
@@ -49,12 +64,25 @@ class TransactionSummaryCharts extends React.Component<Props> {
     });
   };
 
+  handleTrendDisplayChange = (value: string) => {
+    const {location} = this.props;
+    browserHistory.push({
+      pathname: location.pathname,
+      query: {...location.query, trendDisplay: value},
+    });
```

Comment:
If the release series is off, and the current metric is switched the release series re-enables itself. I expected it to stay off.

Reply:
I removed the legend unselectedSeries on purpose as it was causing issues when you unselected `p50`, for example, on duration breakdown will also unselect the p50 of trends. This isn't really what we want, so I either have to change the legend in some way (it's already long), or make yet another query param to not conflict. I felt it was a little heavy, thoughts?

Reply:
Could we drop legend selections for the percentile values for this chart? It is a bit more complex but gives a slightly better experience.

Reply:
You mean only do release series? that's doable. I can see people being fine if releases stays off between charts.

Reply:
I tried ignoring the aggregate selection, but the ux of it isn't that ideal considering you can deselect some but not others, and I think there is probably a case for wanting to deselect the raw data to see the trend. I ended up just making it a separate query param to side step the issue.